### ISSUE TITLE
[Questions] Balance, Combat Reports, and updates

### ISSUE THREAD

**Issue by SabreMania** [tsunamayo/Starship-EVO#3087]
Greetings,

It has been a long while since I posted anything here. The last time I did, it was to make Combat Reports, such as thread #1861 . I will say, much like Dwarf Lord, I have been putting time away from the game due to the Copy and Paste feature being missing. This combined with fewer free time and my usual building speed made me quite absent for some time. However, now that this feature was confirmed to come for the next update, and is actively present in the Experimental branch, I have decided to come back around a little, in order to ask a few questions.

For starters, are you, Tsuna, still looking for Balance suggestions? I recall you asking for us to test how ships faired against each other, what fighting roughly looked like, two months or so before the game became public. As far as I have understood, there were very little changes on this level since last I took off. As such, I must ask, are you still looking for these battle reports, and are you still looking for them to be for ships no bigger than 120-ish meters? 

Second of all, I have no doubt that I am not the only one looking forward to the copy and paste update. Much like everyone else, I would love to know when this update will come, but I understand that bugs need to be done away before it is published. 

Next, I wanted to ask a question related to balance in general. Like I mentionned, I do not believe the game has changed much in terms of raw battle gameplay, other than the AI being less keen on fighting, and more keen on running. Are we going to get a way for us to spawn an AI that will stay and fight, and do you have any idea on what kinds of changes you may want to do soon on ships, weapons, blocks and systems statistics? I reckon that shields, for example, are still far too strong, compared to how much damage most weapons, minus beams, can produce, at least on the level of ships we are used to building.

**Comment by WILDCATreactor** [comment]
Agreed, I have, I think 6 or 12 blocks of shield generators plus 6 capacitors and the regen is 9000/sec, which feels massive without checking against the damage potential of mid-range anti-ship weapons.  Been thinking about how to balance beams against lasers, as well, just to ponder for fun, only thing besides making damage scales comparable is to pull the range on beams down by 30%-50% and increase rate of fire on lasers, especially the cooling the gattling barrels.  Gattling barrels don't even feel comparable to a machinegun atm.

**Comment by tsunamayo** [comment]
Hi, so I have only worked on gimbal weapon and on improving hardpoint as it was the main feedback for a lot of player.
I am not currently focusing on combat but will soon resume as I will work more on NPC spawn as I finish the galaxy update.
The copy-paste is still wip, but should be in default branch too.

Regarding the planned changed, it is mainly about addons:
- Gatling barrel: this one is problematic. I think the only way out is to promote it as a true weapon type, with a size 4x time the standard weapon. So we would have a greater rate of fire and DPS without having to overly nerf range or precision too much (it will still have to be lower), as the base weapon is 4x more powerful. You could plug the other barrel addon on this new gatling gun.
- Side addon: also not satisfied with them, I am currently exploring some way to make it a mechanic more tied to game progression.
- Shield: I will likely tweak those, have a base shield block, plus a shield capacity and a shield recharge addon. It will make more sense.
- Addon stack: I will introduce a rule: max stack = block height, it will make more sense from a balance point of view.
- Lightcruise thruster and fuel: I will do a fuel update for survival. I think it would make more sense to have lightcruise use fuel directly, not power. Power also would use fuel. But basic fuel would be easy to obtain (scoop some fuel around the sun). I am still exploring a few solutions, it is not really related to combat though, just about overall system balance.

Cheers

**Comment by WILDCATreactor** [comment]
Thoughts
Gattling: considering most rotary cannons range from heavy rifle (7.6mm-12.7mm, there's also the new 5.56mm microgun) to light cannon (20-30mm) and those weapons tend to blanket an close area (2-50m at median effective range) in a hail of fire, a combination nerf over damage, range, and precision seems apropos.  Another limitation factor is ammunition storage, which has always plagued this type of weapon.  What if instead of upping the base weapon "rate of reload" alongside rate of fire you had a "shot capacitor" addon that could store additional weapon shots- you'd need to differentiate rate of fire from rate of recharge, but I think doing that would also let you do interesting things with the way the other four barrels work as well.  OR, like you said, the gattling laser base could be a unique one that delink those two values and allows ammunition capacity to be part of the design factor just for them.

Addon Progression: that would imply some addons will be legitimately better in some cases than others, but not necessarilly provide the same cost effectiveness nor the same bonuses and penalties as lower-tier addons.  I.e. T1 Addon gives a +1 bonus to stat A and B and a -1 penalty to C (net +1), costing 10 materials (T1 price model: 10mats per +1); the T2 Addon gives a straight +2 bonus to A, +1 to B, and -1 to D (net +2), costing 25 materials (T2pm: 12.5mt/+1); the T3 Addon gives a +2 bonus to A and C but a -1 to B (net +3), costing 40 materials (T3pm:13.334mt/+1); etc etc.  In this way you can designate certain addons as "more advanced" while still keeping earlier techs viable inside of their niches.

Shield/Reactor Addons: definitely room for expansion, I approve

Addon stack: as in for scalable addon blocks?

Lightcruise Fuel: bueno, both lightcruise and reactors can justifiably use something simple and ubiquitous like Helium as a fuel element as well.  were you also thinking of adding fuel costs for beams and shields?  Second question- solar panels for slowboats and emergency power?
